### PR TITLE
Turn Off Federated Auth and Move Kafka Images to MCR

### PR DESCRIPTION
**Pull request by tvaron3**
# Description

Turned off federated auth for the time being while managed identity work is in progress and changed the kafka images to be stored in the internal microsoft artifact registry.

### CODE REVIEW

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

File: sdk/cosmos/kafka.yml [hunk 1]
```diff
@@ -7,13 +7,14 @@ extends:
       - template: /eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml
         parameters:
           TestName: 'Kafka_TestContainer_Integration'
+          UseFederatedAuth: false
```
Comment (weshaggard) [inline sdk/cosmos/kafka.yml:10]: Why was this turned off? None federated auth will be the only way to work soon.

**Pull request merged**